registrations/models.py

```diff
@@ -39,6 +39,137 @@ class SignUpNotificationType:
     TRANSFERRED_AS_PARTICIPANT = "transferred_as_participant"
 
 
+signup_email_texts = {
```

Comment:
I was thinking that we should probably separate the notification-related code to their own module(s) so that `models.py` could be more model-focused. Maybe create a `notifications.py` and optionally a `notifications_utils.py`, and move `signup_email_texts` and functions like `_get_confirmation_template_texts` to those modules?